MangaManager/ExternalSources/MetadataSources/metadata/AniList.py

```diff
@@ -15,51 +15,79 @@
 
 
 class AniListPerson(StrEnum):
-    OriginalStory = "original_story",
-    CharacterDesign = "character_design",
-    StoryAndArt = "story_and_art",
+    OriginalStory = "Original Story",
+    CharacterDesign = "Character Design",
+    StoryAndArt = "Story & Art",
+    Story = "Story",
+    Art = "Art",
+    Assistant = "Assistant",
+    Assistant_former = "Assistant (former)",
+    Assistant_mob = "Assistant (mob)",
+    Assistant_beta = "Assistant (beta)",
+
+
+class AniListSetting(StrEnum):
+    SeriesTitleLanguage = "series_title_language",
 
 
 class AniList(IMetadataSource):
     name = "AniList"
     _log = logging.getLogger()
     # Map the Role from API to the ComicInfo tags to write
-    person_mapper = {
-        "Writer": [
-            "Writer"
-        ],
-        "Character Design": [
-            "Penciller"
-        ],
-        "Story & Art": [
-            "Writer",
-            "Penciller",
-            "Inker",
-            "CoverArtist"
-        ]
-    }
+    person_mapper = {}
+    aniList_setting = {}
 
     def __init__(self):
         self.settings = [
             SettingSection(self.name, self.name, [
-                SettingControl(AniListPerson.OriginalStory, "Original Story", SettingControlType.Text, "Writer",
+                SettingControl(AniListSetting.SeriesTitleLanguage, "Prefer Romaji Series Title Language",
+                               SettingControlType.Bool, True,
+                               "How metadata field will map to Series and LocalizedSeries fields"),
+                SettingControl(AniListPerson.OriginalStory, AniListPerson.OriginalStory, SettingControlType.Text,
```

Comment:
This is why I had "original_story", "Original Story". The key needs to have _ to the serialization.